tests/acceptance/FormEditorTextInputStylesCest.php

```diff
@@ -32,23 +32,37 @@ public function changeTextInputStyles(\AcceptanceTester $i) {
     $i->fillField('.mailpoet-automation-styles-border-size input[type="number"]', 10); // Set border size
     $i->click('.mailpoet-automation-label-within-input-toggle input'); // Toggle lable to be rendered outside the input
 
-    // Check element has styles
+    $i->wantTo('Check element has styles');
     $i->assertAttributeContains('[data-automation-id="editor_first_name_input"]', 'style', 'border-width: 10px;');
     $i->assertAttributeContains('[data-automation-id="editor_first_name_label"]', 'style', 'font-weight: bold;');
-    // Apply to all
+
+    $i->wantTo('Apply to all');
     $i->click('[data-automation-id="styles_apply_to_all"]');
-    // Check email block has styles too
+
+    $i->wantTo('Add heading block and write some title');
+    $i->addFromBlockInEditor('Heading');
+    $i->fillField('[data-title="Heading"]', 'Lorem Ipsum');
+    $i->see('Lorem Ipsum');
+
+    $i->wantTo('Add paragraph block and write some text');
+    $i->addFromBlockInEditor('Paragraph');
+    $i->fillField('[data-title="Paragraph"]', 'Lorem ipsum dolor sit amet');
+    $i->see('Lorem ipsum dolor sit amet');
+
+    $i->wantTo('Check email block has styles too and save the form');
     $i->assertAttributeContains('[data-automation-id="editor_first_name_input"]', 'style', 'border-width: 10px;');
-    // Save form
     $i->saveFormInEditor();
-    // Reload page and check data were saved
+
+    $i->wantTo('Reload page and check data were saved');
     $i->reloadPage();
     $i->waitForElement('[data-automation-id="form_title_input"]');
     $i->assertAttributeContains('[data-automation-id="editor_first_name_input"]', 'style', 'border-width: 10px;');
     $i->assertAttributeContains('[data-automation-id="editor_first_name_label"]', 'style', 'font-weight: bold;');
     $i->assertAttributeContains('[data-automation-id="editor_first_name_input"]', 'style', 'border-width: 10px;');
+    $i->see('Lorem Ipsum');
+    $i->see('Lorem ipsum dolor sit amet');
```

Comment:
I am not sure those checks will work properly. You are using two texts that only differ in a single letter. I can see two problems with this:

1. Is the method `see` case sensitive? I tried to find it in the documentation, but I couldn't find it mentioned anywhere. 
2. Those texts are quite vague so if we see that the test failed because `'Lorem Ipsum'` was not found we would need to read the whole text to understand what is missing. I think it would be better to add the name of the block in the string. Or even better use the automation id.